### ISSUE TITLE
Defect Report: Noto Sans Arabic (UI): Shadda and Fathatan make a disrubting space

### ISSUE THREAD

**Issue by mrg0lden** [googlefonts/noto-fonts#1559]
### Font 
'NotoSansArabic-Regular.otf' and 'NotoSansArabicUI-Regular.otf`.

### Where the font came from, and when

Site: https://noto-website-2.storage.googleapis.com/pkgs/NotoSansArabic-hinted.zip
Date: 2019-07-04 (preferred format)

### Font Version
2.000

### OS name and version
OS: Linux Ubuntu 19.04 x86_64 
Kernel: Linux 5.0.0-20-generic 
### Application name and version

Noticed on Firefox, LiberOffice Writer, and other programs.

### Issue

  1. Steps to reproduce
    1. Open LiberOffice Writer and set font to Noto Sans Arabic or Noto Sans Arabic UI
    1. type one of these words ريًّا، غيًّا، ربًّا، تبًّا ... 
    1. the last char will look seperated
  2. Observed results
![image](https://user-images.githubusercontent.com/8347078/61245267-c70b2a80-a754-11e9-87e4-81f443322f42.png)

  3. Expected results
  This is how it should look as in Noto Naskh Arabic
![image](https://user-images.githubusercontent.com/8347078/61245478-3ed95500-a755-11e9-9a5e-79da374b6a9e.png)

And this how it looks without a Shadda (ّ  )
![image](https://user-images.githubusercontent.com/8347078/61245572-87910e00-a755-11e9-89c1-b771d7f8d1a2.png)

  4. Additional information 
N/A

### Character data
This word reproduces the problem: 
بًّا

Baa (AKA Beh): U+0628 (ب)
Shadda: U+0651 ( ّ   )
Fathatan: U+064B (ً  )
Alef: U+0627 (ا)

**Comment by iwsfutcmd** [comment]
Just noticed this as well. The fonts that I've seen are affected are:

Noto Sans Arabic 2.000 (all weights and widths, VF and non-VF)
Noto Sans Arabic UI 2.000 (all weights and widths, VF and non-VF)
Noto Kufi Arabic 2.100 (all weights, VF and non-VF)

Noto Naskh Arabic and Noto Nastaliq Urdu are fine, as evidenced below:

![Screenshot from 2019-12-03 16-48-06](https://user-images.githubusercontent.com/12055904/70102571-5f635800-15ed-11ea-91b6-64e4169b3a4e.png)

It appears to only be occurring in combinations of either:
ARABIC SHADDA (U+0651) + ARABIC FATHATAN (U+064B) or
ARABIC FATHATAN (U+064B) + ARABIC SHADDA (U+0651)

This spacing error does not seem to be occurring for any other combinations of Arabic diacritics. A quick check of the fonts themselves shows that it appears the lower anchors on the glyph `shadda_fathahanar` in both fonts were removed at some point, and that might be causing the issue. Similar glyphs `uniFC5E` and `uniFC5F` have multiple anchors below the glyph, whereas `shadda_fathahanar` does not.

**This actually breaks the font completely**, as these character combinations are incredibly common in Arabic text, even occurring commonly in unvocalized texts. I would suggest this be considered high priority.

Cheers,
Ben

**Comment by iwsfutcmd** [comment]
success! I've made a pull request here to fix the issue: https://github.com/googlefonts/noto-source/pull/141

Here's what it looks like after my fix:

![Screenshot from 2019-12-04 04-15-49](https://user-images.githubusercontent.com/12055904/70142002-1f33c200-164d-11ea-9783-3b89ec220829.png)

Once the pull request is accepted and the fonts are rebuilt and uploaded to this repository, the fonts will be fixed.

**Comment by mrg0lden** [comment]
@iwsfutcmd Thank you so much for giving some of your time to fix this. I really appreciate it!

Please notify me once the PR is merged so I close this issue.

**Comment by iwsfutcmd** [comment]
Sure thing! When I noticed it on my own machine I really wanted to fix it.

The pull request was approved, so the fix should at some point propagate down to this repo and then to the website, Google Fonts, etc.

**Comment by mhsabbagh** [comment]
Hey folks.

So this is marked as fixed around a year ago, but I just compiled the latest sources and I still have the bug here:

![2020-09-21_22-20](https://user-images.githubusercontent.com/1293003/93811340-aa238e80-fc58-11ea-9c9a-0e0ca2fbd277.png)


Any idea?

**Comment by punchcutter** [comment]
Since this issue was originally opened there have been some major changes to the file, but the latest sources seem fine to me. Well, except that the Glyphs file won't build with fontmake unless I fix it first. And the zero U+0030 is incorrectly interpolating. The fonts in https://github.com/googlefonts/noto-fonts are also broken so this needs a new issue.
